views/homepage.handlebars

```diff
@@ -1,2 +1,8 @@
 {{!-- just need something in here to render homepage --}}
-<h1> Welcome to the homepage</h1>
\ No newline at end of file
+<div>
+    <img src="/images/rink.jpg" alt="">
+    <div class="motto">
+        <h2>Organize. Stay Current. Get Pumped.</h2>
+    </div>
+    <button><a href="/signup">Create an Account</a></button>
```

Comment:
since this  is a button add the class class="button is-success" so it look look the same as the other singup buttons